docker/jupyterlab/azure-pipelines.yml

```diff
@@ -93,3 +93,36 @@ jobs:
 
         displayName: "Deploying to staging"
         condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/main'))
+
+  - job: deployToProd
```

Comment:
Trenger du egenlig denne?

Reply:
Tror den trengs for å kunne spesifisere hvilken pool jeg vil bruke, jupyterhub-onprem-prod.

Reply:
Jeg mener det scriptet som kjøres i `deployToProd`. Den kjører jo bare en `docker pull $IMAGE` og har så vidt jeg forstår ikke noe med deployment å gjøre

Reply:
Hvis du har tid, kanskje ta en prat på feks teams? Jeg tenker jeg kan få mye ut av et lite møte.